qt/src/hocr/HOCRPdfExporter.cc

```diff
@@ -775,3 +773,13 @@ void HOCRPdfExporter::updateValid() {
 	}
 	ui.buttonBox->button(QDialogButtonBox::Ok)->setEnabled(valid);
 }
+
+void HOCRPdfExporter::addInfoClicked() {
+	Source* source = MAIN->getSourceManager()->getSelectedSources().first();
```

Comment:
Just noticed: This will crash if you don't have a source selected (which in theory can occur if you delete or move the source document, so the attempt by the HOCR editor to automatically open the document will fail, but you are still able to open the export PDF dialog).

Reply:
Ok, what should i do here?

Reply:
QList<Source*> sources = MAIN->getSourceManager()->getSelectedSources();
    if(!sources.isEmpty()) { /* ... */ }